department-of-veterans-affairs/va.gov-cms
# VACMS-3387: Schemata entries pollute dblog.

## Description

See #3387.  A vague attempt to curry favor with Swirt.

I opened an issue and submitted a patch [upstream](https://www.drupal.org/project/schemata/issues/3190131#comment-13950489), then added it to `composer.json`.

## Testing done
Ran `composer install` and observed that the offending statement was removed from SchemaFactory::create().

## Screenshots


## QA steps
As an administrator:
1. Look at the [dblog](/admin/reports/dblog?type%5B%5D=schemata&severity%5B%5D=5) after export.
2. Verify the restoration of a quieter, more peaceful dblog:
- [x] when the cms export is pulled, nothing related to schemata appears in the logs.

### Definition of Done


### Select Team for PR review


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
